test/e2e/tests/account/incremental-security.spec.ts

```diff
@@ -20,6 +20,8 @@ describe('Incremental Security', function (this: Suite) {
         title: this.test?.fullTitle(),
         dappPath: 'send-eth-with-private-key-test',
       },
+      // TODO: Replace `any` with type
```

Comment:
Another change that seems to originate from the Firefox onboarding cherry-pick. This change wasn't in the original PR on `main`, and remains not present on `main`.

Reply:
I see that `main` is using a valid type here instead of `any`, but that's not included in this RC. Seems OK to leave this alone for now, this change should get removed as part of v12.19.0.